### ISSUE TITLE
Japanese `readtime.other` shows a full-width `＃` instead of the correct number

### ISSUE THREAD

**Issue by gh640** [squidfunk/mkdocs-material#7099]
### Bug description

The issue is with the translation of the Japanese text.

When the theme language is set to Japanese (`ja`) and the blog plugin is used, the string `readtime.other` (which is "# min read" in English) is not displayed correctly.

Specifically, the number representing the minutes is not displayed properly, resulting in the text "このページは約＃分で読めます" (which is "＃ min read" in English).

It should actually be displayed as "このページは約15分で読めます" (which translates to "This page can be read in about 15 minutes").

I guess the cause should be the use of a full-width `＃` character where a half-width `#` character should be used.

(This is an translation issue. I apologize if I am mistaken about the place and method to report this.)

### Related links

- https://github.com/squidfunk/mkdocs-material/issues/4944
- https://squidfunk.github.io/mkdocs-material/contributing/adding-translations/

### Reproduction

[9.5.18-wrong-number-display-in-japanese.zip](https://github.com/squidfunk/mkdocs-material/files/15102381/9.5.18-wrong-number-display-in-japanese.zip)

### Steps to reproduce

1. Run the dev server by `mkdocs serve`
2. Open the blog post page in a browser: http://localhost:8000/blog/2024/04/24/this-is-a-sampple-post/
3. Check the display of `ul.md-post__meta` element

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) using the [built-in info plugin](https://squidfunk.github.io/mkdocs-material/plugins/info/).

**Comment by gh640** [comment]
## Screenshots

English (correct):

![Screenshot 2024-04-25 at 9 11 34](https://github.com/squidfunk/mkdocs-material/assets/3532528/00a1f7bf-bf17-4051-850b-3f0744480127)

Japanese (incorrect):

![Screenshot 2024-04-25 at 9 06 09](https://github.com/squidfunk/mkdocs-material/assets/3532528/5a81844f-d95e-4bf8-84d8-286db6614450)

**Comment by gh640** [comment]
By the way, currently the translation for `readtime.one` is "このページは約１分で読めます" ("This page takes about 1 minute to read"). The "１" here is a full-width character. 

But for numbers greater than 1 (in the case of `readtime.other`), a half-width character is used. To improve consistency, I think it would be better to use a half-width "1" for `readtime.one` as follows:

> このページは約1分で読めます

**Comment by squidfunk** [comment]
Thanks for reporting! Fixed in bde889fab1d9ce05cc83de55a6f09fb4f4f3c4e9.

Yes, we're currently not able to use full-width numbers, so switching to half-width to be consistent is likely to be our best option. We could add whitespace left and right, if that makes reading any easier, but then we should also adjust that for:

``` py
  "search.result.more.one": "このページ内にもう1件見つかりました",
  "search.result.more.other": "このページ内にあと#件見つかりました",
```

We'll be working on improving internationalization in the future, as we want to provide the best experience possible!

**Comment by gh640** [comment]
@squidfunk 

Thank you for your quick response! I have confirmed that the issue has been resolved by running a sample with `master` after bde889f :D

> We could add whitespace left and right, if that makes reading any easier, but then we should also adjust that for:

You are right. FYI, whether or not to add spaces around half-width characters within Japanese sentences is a point of contention. My preference is to add spaces, but there are also people who don't like it. Even within a single company, there is no consistency. For example, spaces are added in the UI within macOS, but not on Apple's website (apple.com).

If there is an opportunity in the future when internationalization is improved, I would love to contribute.

I will close this issue. If there are any problems, please reopen it. Thanks :)

**Comment by squidfunk** [comment]
> You are right. FYI, whether or not to add spaces around half-width characters within Japanese sentences is a point of contention. My preference is to add spaces, but there are also people who don't like it. Even within a single company, there is no consistency. For example, spaces are added in the UI within macOS, but not on Apple's website (apple.com).

Thanks for the clarification! Then let's just leave it like that for now, as [translations can also be quite simply overridden](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#custom-translations). I'll let you know when we start working on better translations support ✌️

Let's keep this issue open until released ☺️

**Comment by squidfunk** [comment]
Released as part of [9.5.19](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.19)

**Comment by gh640** [comment]
Yay! Thank you.

Yes, please reach out when the time comes ✌️
